bootstrap/cmd/bootstrap/app/server.go

```diff
@@ -45,19 +45,63 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"io/ioutil"
 )
 
 // RecommendedConfigPathEnvVar is a environment variable for path configuration
 const RecommendedConfigPathEnvVar = "KUBECONFIG"
 
-// DefaultStorageAnnotation is the name of the default annotation used to indicate
+// DefaultStorageAnnotation is the Name of the default annotation used to indicate
 // whether a storage class is the default.
 const DefaultStorageAnnotation = "storageclass.beta.kubernetes.io/is-default-class"
 
 // Assume gcloud is on the path.
 const GcloudPath = "gcloud"
 
-const Kubectl = "/usr/local/bin/kubectl"
+const RegistryName = "kubeflow"
+
+type KsComponent struct{
+	Name      string
+	Prototype string
+}
+
+type KsPackage struct{
+	Name string
+}
+
+type KsParameter struct{
+	Component string
+	Name      string
+	Value     string
+}
+
+//
+type AppConfig struct {
+	Packages   []KsPackage
+	Components []KsComponent
+	Parameters []KsParameter
+}
+
+type BootConfig struct {
```

Comment:
Why are there multiple apps? And if you want to support multiple apps why wouldn't you use a container like a list or map.

My sense though is we should start with a single app unless we have a clear use case for multiple ones.

I think if a user wanted multiple apps they could run bootstrapper multiple times